lib/index.js

```diff
@@ -57,9 +57,11 @@ function init() {
           if(tmp in schemas) {
             logger.debug('overwriting schema "' + tmp + '" with ' + file);
           }
-          schemas[tmp] = module[key];
-          schemas[tmp].instance = schemas[tmp]();
-          logger.debug('loaded schema: ' + tmp);
+          if(typeof module[key] === 'function') {
```

Comment:
what happens if schema is not a function?

Reply:
look at the next line, the issue is if we pass a schema that is not a function the current setup assumes it is a function.


https://github.com/digitalbazaar/bedrock-validation/blob/145c388b372832b39abdc408891f45a87e868d4b/lib/index.js#L55-L63

code in master fails if schema is an object right now.
I also might add the current way this works is pretty bad. a naming convention like validator.schema.js would be better so we could at least have some control over which files in a dir are inspected for schemas.

Reply:
The question I meant to ask was:  What happens **after this patch** if schema is not a function?

Reply:
it is ignored.